backend/kesaseteli/applications/models.py

```diff
@@ -100,6 +104,13 @@ class YouthApplication(HistoricalModel, TimeStampedModel, UUIDModel):
     receipt_confirmed_at = models.DateTimeField(
         null=True, blank=True, verbose_name=_("timestamp of receipt confirmation")
     )
+    encrypted_vtj_json = EncryptedCharField(
+        null=True,
+        blank=True,
+        max_length=1024 * 1024 * 256,
```

Comment:
Can it really be this large?

Reply:
> Looks good to me! The length limit in the encrypted_vtj_json seems quite large, if you feel it could be shorter then I suggest changing it, but merge at will

I lowered the max_length of the encrypted_vtj_json from 256MiB to 1MiB.

BTW YouthApplication model is historized using HistoricalModel which uses HistoricalRecords from django-simple-history and looking at its [documentation](https://django-simple-history.readthedocs.io/en/latest/quick_start.html#what-is-django-simple-history-doing-behind-the-scenes) it says—here Choice and Poll are models whose changes are historized:
> Every time a create, update, or delete occurs on Choice or Poll a new row is created in the historical table for that model including all of the fields in the instance of the base model

So, having the encrypted_vtj_json duplicated every time when any of the fields in YouthApplication change will make the database blow up quite easily if the field is large. I won't address this in this PR but I will make a ticket about this.

Reply:
Most likely not, see https://github.com/City-of-Helsinki/yjdh/pull/569#issuecomment-1011997948